backend/src/openarchiefbeheer/destruction/api/serializers.py

```diff
@@ -145,8 +150,8 @@ def update(
             instance.bulk_create_items(items_data)
 
         if assignees_data is not None:
-            instance.assignees.all().delete()
```

Comment:
If record manager updates list and status is in review, reject the change with a 403 and indicate that the list can not be changed due the review already being started. 

List should be able to be changed when status is "changes requested" (see: Jira)

Reply:
As discussed: We "probably" need a status "new" which should allow changes. So only accept changes to the list if the status is "new" (new can be named differently)